src/ContainerDetails.jsx

```diff
@@ -43,6 +44,17 @@ const ContainerDetails = ({ container }) => {
                         <DescriptionListDescription>{utils.quote_cmdline(container.Config.Cmd)}</DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    {Boolean(container.Config?.Labels?.PODMAN_SYSTEMD_UNIT) && (container.uid === 0 || container.uid === null) &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm>{_("Systemd service")}</DescriptionListTerm>
```

Comment:
It's "systemd", not "Systemd". https://systemd.io/ doesn't capitalize it either.

Reply:
:100: carelessly copied from the design.